client/src/sections/Login/Login.tsx

```diff
@@ -1,17 +1,27 @@
-import React from 'react';
+import React, {useState, useCallback} from 'react';
 import {useHistory} from 'react-router-dom';
 import {Button, Card, Heading, ButtonGroup} from '@shopify/polaris';
-import logo from 'icons/logo.jpg';
+import logo from 'icons/logo.png';
 
 import './Login.css';
+import LoginModal from './LoginModal';
 
 export default function Login() {
   const history = useHistory();
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+  const [active, setActive] = useState(false);
```

Comment:
We should try to make states more meaningful. What is active here? I think a better name for the state would be `showModal`.